controllers/app-controllers.js

```diff
@@ -37,3 +38,16 @@ exports.getCommentsByReviewId = (request, response, next) => {
     })
     .catch(next);
 };
+
+exports.postCommentOnReviewId = (request, response, next) => {
+  const { review_id } = request.params;
+  const updates = request.body;
```

Comment:
It would be great if this was named something a little more descriptive like "newComment".